docker-compose-eth-ganache.yaml

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  tcf:
+    container_name: tcf
+    image: tcf-dev
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.tcf-dev
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+        - MAKECLEAN=${MAKECLEAN:-1}
+        - TCF_DEBUG_BUILD=${TCF_DEBUG_BUILD:-}
+        - DISPLAY=${DISPLAY:-}
+        - XAUTHORITY=~/.Xauthority
+    volumes:
+      # Below volume mappings are required for DISPLAY settings by heart disease GUI client
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - ~/.Xauthority:/root/.Xauthority
+      - ./examples:/project/TrustedComputeFramework/examples
```

Comment:
You could avoid volume mapping of examples directory. I assume you added it for debugging.